test/index.js

```diff
@@ -187,7 +194,7 @@ function testFile(engine, file, filename, index) {
     }
   }
 
-  console.log('#%d. %s completed.', index, filename);
+  console.log('    #%d passed (%s)', index, filename);
```

Comment:
Like where you're going. Sorry for not being explicit in what the brain-hiccup was.

The double digit is what made it difficult to read for me.

```
52. beginning something something
52. something something completed
```

So getting rid of the digit.

```
52. beginning something something
failed
```

And

```
52. beginning something something
failed
```

Might be easier. Maybe even adding a blank line after each group??

```
52. beginning something something
failed

53. beginning something something
passed
```